src/Network/Send/cRO.pm

```diff
@@ -115,4 +123,26 @@ sub sendCharCreate {
 	$self->sendToServer( $msg );
 }
 
+# 0x0436,23
+sub sendMapLogin {
+	my ($self, $accountID, $charID, $sessionID, $sex) = @_;
+	my $msg;
+	$sex = 0 if ($sex > 1 || $sex < 0); # Sex can only be 0 (female) or 1 (male)
+
+	#my $unknown = pack('C4', (0x34, 0x0B, 0x01, 0x00));
+
+	$msg = $self->reconstruct({
+		switch => 'map_login',
+		accountID => $accountID,
+		charID => $charID,
+		sessionID => $sessionID,
+		unknown => 68404,# 34 0B 01 00
```

Comment:
I got `AB 0B 01 00`

Reply:
maybe has changed, from discord:
![image](https://user-images.githubusercontent.com/10372732/131270545-23b5f21e-c4c1-4308-839f-77e6ada3a807.png)

Reply:
it seems the server don't check these unknown 4 bytes , bot will log in as long as recvpackets.txt is right.

Reply:
@hubertyang88, what values do you have now?

Reply:
I use this one '68404'  , but I tried 0 or 77777 or 99999, all work , I guess server don't care this data , at least for now

Reply:
can you intercept packets from the client?

Reply:
36 04 3d f3 01 00 c4 bc 05 00 06 03 00 00 26 0b 01 00 86 bf 9b 01 00

Reply:
i am getting now
36 04 AC 1B 03 00 DC CE 04 00 2D 34 00 00 BF 0B 01 00 D7 23 7F 2A 00
here is a constant: `0B 01 00`

ok, let's leave this part of the code unchanged